addons/portal/controllers/portal.py

```diff
@@ -181,10 +181,77 @@ def home(self, **kw):
         values = self._prepare_portal_layout_values()
         return request.render("portal.portal_my_home", values)
 
+    @route(['/my/account/save'], type='http', auth='user', website=True)
+    def save_address(self, **kw):
+        address = int(kw.get('address')) if kw.get('address') else False
```

Comment:
Rather than converting it into integer like this we should convert value in route like this:
```suggestion
    @route(['/my/account/save/<int:address>'], type='http', auth='user', website=True)
    def save_address(self, **kw):
```
I haven't tested the code please test it and apply changes if anything needs to change

Reply:
Hello @kpt-odoo 

changes are done :slightly_smiling_face: .